tests/test_runner.py

```diff
@@ -1,3 +1,5 @@
+from buildrunner.docker.daemon import DAEMON_IMAGE_NAME
+from buildrunner import docker as buildrunner_docker
```

Comment:
The ordering of these imports seems strange to me. But Ruff forced me to put these at the top.

Reply:
Uhh, that's definitely wrong, why would ruff tell you that?

Reply:
`# noqa: E402` to the rescue.